packages/components/.eslintrc.js

```diff
@@ -1,19 +1,22 @@
 module.exports = {
-    "extends": "@mapsindoors/eslint-config-typescript",
-    "parser": "@typescript-eslint/parser",
-    "env": {
-        "browser": true,
-        "es6": true,
-        "jest": true
+    'extends': '@mapsindoors/eslint-config-typescript',
+    'parser': '@typescript-eslint/parser',
+    'env': {
+        'browser': true,
+        'es6': true,
+        'jest': true
     },
-    "globals": {
-        "google": "readonly"
+    'globals': {
+        'google': 'readonly'
     },
-    "parserOptions": {
+    'parserOptions': {
         ecmaVersion: 2018,  // Allows for the parsing of modern ECMAScript features
         sourceType: 'module',  // Allows for the use of imports
     },
-    "rules": {
-        "require-await": 0 // Stencil requires publicly exposed methods to be async, but has no requirement to implement any asynchronous code
+    'rules': {
+        'require-await': 0, // Stencil requires publicly exposed methods to be async, but has no requirement to implement any asynchronous code
+        '@typescript-eslint/no-unused-vars': ['error', {
+            'varsIgnorePattern': '^h$'
```

Comment:
A comment explaining why we ignore `h` would be nice to the future web team.

Reply:
👍